### PR TITLE
Enhance/update tracking events

### PR DESCRIPTION
**Pull request by yanglin5689446**
## Summary

events list:
https://docs.google.com/spreadsheets/d/1tk1sEZfJn8xcMX8-V_iArjoEDiUUVLUJNPRomUwg6XU/edit#gid=10975643

## Related Links

**Asana**:  

**Mockup**:  

## Checklist

- [ ] Pasted Asana or Mockup link.
- [ ] Tagged labels.

### CODE REVIEW

**Comment by yanglin5689446** [comment]
> Would you attach the link to the table of events that is updated here to this pull request?
updated

File: src/services/Amplitude/log.ts [hunk 1]
```diff
@@ -2,9 +2,20 @@ import { getInstance } from "./system";
...
 const IS_LOCAL =
   process.env.REACT_APP_ENV === "local" || !process.env.REACT_APP_ENV;
-const logCore = (name: string, properties = {}, callback = () => undefined) => {
+
+const logCore = (
+  name: string,
+  rawProperties: { [key: string]: unknown } = {},
+  callback = () => undefined
+) => {
+  // strip undefined fiels
```
Comment (moojing) [inline src/services/Amplitude/log.ts:11]: is this `fields`?

**Review by moojing — APPROVED** [review]

**Comment by moojing** [comment]
Do we have any plan to remove old events after v2.0 is online or the old wallet is retired?

File: src/services/Amplitude/log.ts [hunk 2]
```diff
@@ -18,100 +29,125 @@ const logCore = (name: string, properties = {}, callback = () => undefined) => {
...
-  logCore(
-    "web_sdk_sign_tx",
-    dAppId ? { domain, chain, url, dAppName, dAppId } : { domain, chain, url }
-  );
+  logCore("wallet_approve_signature", {
+    product,
+    domain,
+    dApp_name: dAppName,
+    dApp_app_id: dAppId,
+    chain,
```
Comment (mordochi) [inline src/services/Amplitude/log.ts:152]: The `chain` param on the spread sheet is capitalized but the `blockchain` param in our app here is all lowercase. Will this affect how the PM retrieves the data?

Comment (yanglin5689446) [reply src/services/Amplitude/log.ts:152]: had checked with faus no worry la ser

File: src/services/Amplitude/log.ts (hunk 1 above)
Comment (yanglin5689446) [reply src/services/Amplitude/log.ts:11]: true dat

**Pull request merged**